http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpAppDescriptorDAO.scala

```diff
@@ -32,19 +28,12 @@ class HttpAppDescriptorDAO[F[_]](httpClient: Client[F])(implicit logger: Structu
 
   private[dao] def resolveUri(path: String)(implicit ev: Ask[F, AppContext]): F[String] =
     for {
-      resp <- fs2.Stream
-        .retry(httpClient.expectOr[String](
-                 Request[F](
-                   method = Method.GET,
-                   uri = Uri.unsafeFromString(path)
-                 )
-               )(onError(path)),
-               5 seconds,
-               delay => delay * 2,
-               5,
-               e => true)
-        .compile
-        .lastOrError
+      resp <- httpClient.expectOr[String](
```

Comment:
+1 on this, `http4s` already retries internally I think.

Reply:
we set up retry in [here](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala#L353) for all calls through http4s

Reply:
Hmm... interesting, I've seen this fail before even with that http4s config which is why I added this here a while ago